### ISSUE TITLE
Bones from skin files have incorrect armature setup

### ISSUE THREAD

**Issue by Kjasi** [Markemp/Cryengine-Converter#73]
When importing character models, such as `Objects\Characters\Human\male_v7\armor\ccc\m_ccc_bear_helmet_01.skin`, `m_ccc_medium_armor_arms_01.skin`, `m_ccc_medium_armor_core_01.skin`, and `m_ccc_medium_armor_legs_01.skin`, they import with an incorrect armature.

This is especially noticeable with Blender's Auto Connect and Chain Bones options enabled. All the bones, which from the names appear to the be a whole biped, appear in a kind of hemisphere, curving along the model's back.
![image](https://user-images.githubusercontent.com/5256123/126478776-3350db6f-4ceb-4334-ad64-136aeb1fd1aa.png)

**Comment by Markemp** [comment]
Can you test out the v1.3.1 draft release against this file?

**Comment by Markemp** [comment]
Issue opened up with Blender.

https://developer.blender.org/T90146

TL;DR:  `Auto Connect` appears to be bugged.  That may not be the issue here, but it's worth being tracked.  Leaving this issue open for further testing.

**Comment by Kjasi** [comment]
> Can you test out the v1.3.1 draft release against this file?
> 
> https://github.com/Markemp/Cryengine-Converter/releases
I don't see any draft releases... Last file I see is 1.3.0. Maybe I need a permission higher than "general public" to see it? Not sure.

**Comment by Markemp** [comment]
The interface isn't very intuitive.  Did you click on the Releases tab?

![image](https://user-images.githubusercontent.com/12057128/126997265-58903652-108a-40bf-8f64-c8e3fda04c4a.png)

**Comment by Kjasi** [comment]
Yes, but I don't see it. 
![image](https://user-images.githubusercontent.com/5256123/126997559-01aa1c27-1154-4800-92cb-661a8acbe457.png)

**Comment by Kjasi** [comment]
Okay, I compiled from source to test it: No meshes are exported anymore, and the skeleton is still in a non-humanoid state: 
![image](https://user-images.githubusercontent.com/5256123/126999729-fed50477-d26a-4595-913d-0271a6f87901.png)

**Comment by Kjasi** [comment]
SKIN files still hang with 1.3.1 Preview, but CGF files with skin files will convert with bones and weights. However, the skeleton still has incorrect positioning when imported:
![image](https://user-images.githubusercontent.com/5256123/127335039-a0dccf03-1f1a-4555-8c11-06c2ab8d8274.png)

**Comment by Markemp** [comment]
This should be fixed on v1.5.0.  The bear head model isn't parenting right, so will go take a look at that too.

**Comment by Markemp** [comment]
Closing this issue as it's resolved in the current release.
